packages/base/src/formbuilder/objectform/baseform.tsx

```diff
@@ -175,6 +177,7 @@ export class BaseForm extends React.Component<IBaseFormProps, IBaseFormStates> {
                 const sliderValue = parseFloat(target._value); // Slider value is in 0–10 range
                 const normalizedValue = sliderValue / 10; // Normalize to 0.1–1 range
                 props.onChange(normalizedValue);
+                syncData({ ...props.formData, opacity: normalizedValue });
```

Comment:
This wouldn't only apply the opacity but all other properties. It feels weird doing an automatic "Ok" for all properties only when moving the opacity slider.

Instead, I'd look into removing the "Ok" button for now and make all properties automatically submit in the object properties form (calling `submit` automatically `onChange` event on the form).

Reply:
I integrated this suggestion in the following commits below